### PR TITLE
[MIG] printer_zpl2: Migration to 12.0

### PR DESCRIPTION
**Pull request by fmdl**
@sla-subteno-it

### CODE REVIEW

**Comment by etobella** [comment]
I made a PR that affected printer_zpl2 and it may be interesting to include it on the migration
#172

**Comment by fmdl** [comment]
@etobella add

File: printer_zpl2/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright (C) 2016 SUBTENO-IT (<https://subteno-it.fr>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Printer ZPL II',
+<<<<<<< HEAD
+    'version': '12.0.1.0.0',
+=======
+    'version': '11.0.1.0.1',
+>>>>>>> 6871638... [FIX] printer_zpl2: QR as bytes
```
Comment (etobella) [inline printer_zpl2/__manifest__.py]: Error on the rebase of commits

**Comment by etobella** [comment]
BTW, great to add the copy on the components :+1:

**Comment by pedrobaeza** [comment]
Please rebase after dependencies merging

**Review by sla-subteno-it — APPROVED** [review]

**Comment by fmdl** [comment]
@etobella could you review ? Regards

**Review by etobella — APPROVED** [review]

**Comment by etobella** [comment]
Review the Ci and runbot please

**Comment by fmdl** [comment]
@etobella it fail because:
- the lint check an other module
- runbot doesn't work because the dependencie zpl2 is not installed

**Comment by etobella** [comment]
Then it's perfect for me

**Review by sla-subteno-it — APPROVED** [review]

**Comment by etobella** [comment]
@fmdl just two things:
There is an issue on line 48 of the readme: https://travis-ci.org/OCA/report-print-send/jobs/532044313#L802 Can you check?
Also, you need to add a file (requirements.txt) with the same information of 11.0 (https://github.com/OCA/report-print-send/blob/11.0/requirements.txt)
With this, all should be green and we could merge. Can you try it please?

**Comment by etobella** [comment]
Now it seems that all is green 
:green_apple::green_apple::green_apple::green_apple::green_apple: 

It should be ready to merge :smile:

**Review by jarroyomorales — APPROVED** [review]
LGTM :+1:

**Pull request merged**